.github/workflows/destroy.yml

```diff
@@ -1,71 +1,41 @@
-name: Destroy Review Instance
+name: Delete review app
+
 on:
   pull_request:
-    types: [closed]
-
-permissions:
-  id-token: write
-  pull-requests: write
-  contents: write
+    types: [closed, unlabeled]
+  workflow_dispatch:
+    inputs:
+      pr_number:
+        description: PR number of review app to delete
+        required: true
+        type: string
 
 jobs:
-  destroy:
-    name: Destroy
-    environment:
-       name: review
+  delete-review-app:
+    name: Delete review app ${{ github.event.pull_request.number }}
     runs-on: ubuntu-latest
-    concurrency: review_${{github.event.number}}
-    defaults:
-      run:
-        shell: bash
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-
-      - name: Set Environment variables
-        shell: bash
-        run: |
-          tf_vars_file=terraform/aks/config/review.tfvars.json
-          terraform_version=$(awk '/{/{f=/^terraform/;next}f' terraform/aks/terraform.tf | grep -o [0-9\.]*)
-          echo "TERRAFORM_VERSION=$terraform_version" >> $GITHUB_ENV
+    concurrency: review_${{ github.event.pull_request.number || github.event.inputs.pr_number }}
+    if: >
+      github.event.action == 'closed' && contains(github.event.pull_request.labels.*.name, 'deploy') ||
```

Comment:
se doesn't use a deploy label. Review apps are built if `github.event_name == 'pull_request' && github.ref != 'refs/heads/master' ` so you'll have to change this to match